### DISCUSSION TITLE
Help me in choosing the path

### DISCUSSION THREAD

**Discussion by ghost** [community/community#77963] [Programming Help] [answered]
### Body

Hi everyone I'm @Yuheshpandian. I'm currently learning html and css. And I decided to learn Web Development. Can Anyone help me in one thing. I recently was informed about UI/UX

So I don't know  whether to learn UI/UX or continue in learning the HTML and CSS.

Please someone help me

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by robskaar** [comment, marked as answer]
In my experience, id say get the fundamentals of a subject right, then practice it, then expand.

So i'd say start with your HTML, CSS And add JS + TS. 
Once these are like second nature to you, expand with learning UX,Git,DevOps, or whatever else your interested in 👍🏼

Context about me:
Fulltime employed webdev 2 years, CS graduate.


Good luck on your journey.

**Reply by ghost** [reply]
Thank you so much for the guidance

**Comment by Hugoalv79** [comment]
Hi @Yuheshpandian  , I recommend you the course of https://www.coursera.org/professional-certificates/google-ux-design and the Front-end professional certificate offered by META. Exists video tutorials of good chanels in Youtbe as freeCodeCamp.org,etc but you can also try paid courses in Udemy, just sort them by quality in stars and hours.  The W3school website is algo a good place to learn.

You can ask for financial aid in Coursera if you need it, regards!

**Reply by ghost** [reply]
Thank you for your great suggestion I will look out these for sure.

**Comment by 8279824760** [comment]
G

**Reply by ghost** [reply]
you mean gravitational constant some other G?
